### ISSUE TITLE
"jq: command not found" for failed jobs

### ISSUE THREAD

**Issue by jackbutler** [CircleCI-Public/jira-connect-orb#46]
### Orb version

1.1.7

### What happened

When a job fails, the "Update status in Atlassian Jira" step also fails:
![image](https://user-images.githubusercontent.com/14986475/84640781-0fe1db00-aef2-11ea-90dc-2c852866c148.png)

The output inside the failing step ends with this:

```
environment: line 27: jq: command not found

Exited with code exit status 127
CircleCI received exit code 127
```

Jira is not updated with the failed build status.

It works as expected when the previous steps in the job all pass, this appears to only be an issue when running `jira/notify` after failed steps.

### Expected behavior

Jira is updated with the failing build status

**Comment by jackbutler** [comment]
I can see in successful cases there is an `install jq` step, that isn't there in failure cases:

**Success**
![image](https://user-images.githubusercontent.com/14986475/84642406-533d4900-aef4-11ea-9ce1-853bde9b2bc3.png)

**Failure**
![image](https://user-images.githubusercontent.com/14986475/84642430-5d5f4780-aef4-11ea-8d1c-8efa3c238c35.png)

**Comment by jnpwebdeveloper** [comment]
Getting the same here. Using `circleci/jira@1.1.7` orb.

**Comment by gmemstr** [comment]
@jackbutler @jnpwebdeveloper Would you mind providing a link to one of the builds in question, or providing the output of the install jq step?

**Comment by jackbutler** [comment]
Sorry I can't provide a link to the build itself, and I would provide the output but the "install jq" step doesn't seem to appear at all for the failed job - this is the full set of steps that I'm seeing:

![image](https://user-images.githubusercontent.com/14986475/86473012-081a9700-bd38-11ea-85e1-129e7051882f.png)

For context it's using `jira/notify` passed as a post-step to the `cypress/run` command from the cypress orb

**Comment by jnpwebdeveloper** [comment]
Yea I get the same as above - also after a failed cypress test.
This is what I get at the end of the update status job - also using the `- jira/notify`. Here is my full workflow for the end to end test we're running:
```
version: 2.1
orbs:
  node: circleci/node@2.0.2
  cypress: cypress-io/cypress@1.24.0
  browser-tools: circleci/browser-tools@1.0.1
  jira: circleci/jira@1.1.7
  jq: circleci/jq@2.1.0

... jobs etc...
workflows:
  end-to-end-test:
    jobs:
      - cypress/run:
          name: Full end to end regression test
          executor: cypress/browsers-chrome77
          browser: chrome
          start: npm start
          wait-on: "http://localhost:8080"
          store_artifacts: true
          no-workspace: true
          post-steps:
            - jira/notify
```

![image](https://user-images.githubusercontent.com/1742908/86474084-0520a600-bd3a-11ea-83f0-64b981942525.png)

**Comment by gmemstr** [comment]
Interesting - thanks for the info, digging into this further!

**Comment by gmemstr** [comment]
Initial findings suggest this isn't an issue with the orb itself, but rather how the configuration parsing is handled within the platform itself. Escalating this internally to try and get to the bottom of this :)

**Comment by gmemstr** [comment]
Final update - looks like it is in the orb! The `jq/install` command will only execute if previous jobs succeeded, but the `jira/notify` command is set to run `when: always`. Should have a fix available soon :)

**Comment by gmemstr** [comment]
This has been fixed as of version 1.2.2.
